src/Messages/KlaviyoTrackMessage.php

```diff
@@ -34,4 +34,4 @@ public function __construct(string $event, array $customer_properties, array $pr
 			$this->properties = $properties;
 			$this->time = $time ? $time->format("U") : null;
```

Comment:
We should be able to get the date format constant from DateTimeInterface which is already included too.

```suggestion
			$this->time = $time ? $time->format(DateTimeInterface::W3C) : null;
```